include/ofi_util.h

```diff
@@ -801,11 +801,14 @@ void util_put_peer(struct util_peer_addr *peer);
  * A future cleanup would be to remove using the util_av and have the
  * rxm_av implementation be independent.
  */
- struct rxm_av {
+struct rxm_av {
 	struct util_av util_av;
 	struct ofi_rbmap addr_map;
 	struct ofi_bufpool *peer_pool;
 	struct ofi_bufpool *conn_pool;
+	struct fid_peer_av peer_av;
+	struct fid_av *util_coll_av;
+	struct fid_av *offload_coll_av;
```

Comment:
This offload changes looks misplaced (here and also below).  Looking ahead, maybe just update the commit message that the changes also add support for offloaded collectives.  There's really 2 sets of independent changes in this commit, but it would take a bit of work to separate them into 2 patches.

Reply:
These are part of rxm provider changes. It looks out of place only because rxm_av.c was moved from /pro/rxm into /prov/util.

Reply:
The commit message says that the change is to separate the util_coll into an offload provider.  However, paired with that change is the additional change to support another offload provider, unless I'm reading the changes wrong.

Reply:
I misread your first comment. I will update the commit message.